internal/hmyapi/backend.go

```diff
@@ -122,31 +122,31 @@ func GetAPIs(b Backend) []rpc.API {
 			Public:    true, // FIXME: change to false once IPC implemented
 		},
 		{
-			Namespace: "hmyv2",
+			Namespace: "hmy_v2",
```

Comment:
Will anyone be affected by this ? Will trustwallet be affected ?

Reply:
Talked with Vlad on discord and he ways trustwallet does not use this at the moment